index.html

```diff
@@ -2,188 +2,233 @@
+                <hr />
+            </div>
+            <div class="portfolio-image works">
+                <img src="/images/ImgPlaceholder.png" alt="" />
+                <div class="multi-post">
+                    <h3>Multi-Post Stories</h3>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. has been the industry's standard dummy text
+                        ever since the 1500s, when an unknown printer took a standard
+                        dummy text.
+                    </p>
                     <ul>
+                        <li>CSS</li>
                         <li>HTML</li>
                         <li>Bootstrap</li>
                         <li>Ruby</li>
                     </ul>
+                    <button class="btn">See Project</button>
                 </div>
-                <button>See project</button>
             </div>
 
-            <div class="card">
-                <div class="card-content">
-                    <h2>Profesional Art Printing Data</h2>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                        industry's standard.</p>
-                    <ul>
-                        <li>HTML</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
+            <div class="my-cards">
+                <div class="card">
+                    <div class="card-content hide">
+                        <img class="first-image" src="/images/Img Placeholder.png" alt="" />
+
+                        <button class="btn-show">See project</button>
+                    </div>
                 </div>
-                <button class="btn">See project</button>
-            </div>
 
-            <div class="card">
-                <div class="card-content">
-                    <h2>Profesional Art Printing Data</h2>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                        industry's standard.</p>
-                    <ul>
-                        <li>HTML</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
+                <div class="card bg-2">
+                    <div class="card-content">
+                        <h2>Professional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
```

Comment:
- [optional] please like the previous reviewer noted, the first card is not alinged with the others like in the original version. remember this cards will represent projects and you wont want you projects to get mixed up. kindly consider implementing this.
![Screenshot (950)_LI](https://user-images.githubusercontent.com/65298048/230737312-1ef912da-e8fd-4a4e-b014-7d49db6e5b47.jpg)

Reply:
From my end, its as below. 
[https://prnt.sc/cUJXvkCgSYGb](url)

How do replicate the above screen?